### PR TITLE
add atresplayer channels

### PR DESCRIPTION
**Pull request by karlo2105**
Spanish channels of Atresmedia group.

### CODE REVIEW

**Comment by beardypig** [comment]
Awesome! Looks like I need to fix the build checking for PRs... :D 
Couple of syntax error I spotted, lemme just annotate them.

Wooo thanks! 🥇

File: atresplayer.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "atresplayer.com",
+  "attributes": {
+    "geolocked": true,
+    "region": "ES",
+    "language": "esp"
```
Comment (beardypig) [inline atresplayer.json]: There is a missing `,` on the end of the line 
```json
"language": "esp",
```

File: atresplayer.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "attributes": {
+    "geolocked": true,
+    "region": "ES",
+    "language": "esp"
+    "hd": true
+  },
+  "streams": [
+    {"name": "antena3",
+      "title": "Antena 3",
+      "url": "http://www.atresplayer.com/directos/television/antena3/"},
```
Comment (beardypig) [inline atresplayer.json]: There shouldn't be a `}` in this line:
```json
"url": "http://www.atresplayer.com/directos/television/antena3/",
```

File: atresplayer.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      "attributes": {
+        "geolocked": false}},
+    {"name": "lasexta",
+      "title": "La Sexta",
+      "url": "http://www.atresplayer.com/directos/television/lasexta/"},
```
Comment (beardypig) [inline atresplayer.json]: Same here, an extra `}` getting in the way :)
```json
"url": "http://www.atresplayer.com/directos/television/lasexta/",
```

File: atresplayer.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      "url": "http://www.atresplayer.com/directos/television/neox/"},
+    {"name": "nova",
+      "title": "Nova",
+      "url": "http://www.atresplayer.com/directos/television/nova/"},
+    {"name": "mega",
+      "title": "Mega",
+      "url": "http://www.atresplayer.com/directos/television/mega/"},
+    {"name": "atreseries",
+      "title": "Atreseries",
+      "url": "http://www.atresplayer.com/directos/television/atreseries/",
```
Comment (beardypig) [inline atresplayer.json]: Here there is a missing `}`, and the `,` is not required :)
```json
"url": "http://www.atresplayer.com/directos/television/atreseries/"}
```

**Review by beardypig — CHANGES_REQUESTED** [review]
A couple of typos :) Otherwise looks good :D

Thanks a lot :)

File: datafiles/streams/atresplayer.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  "attributes": {
+    "geolocked": true,
+    "region": "ES",
+    "language": "esp",
+    "hd": true
+  },
+  "streams": [
+    {"name": "antena3",
+      "title": "Antena 3",
+      "url": "http://www.atresplayer.com/directos/television/antena3/"},
```
Comment (beardypig) [inline datafiles/streams/atresplayer.json]: Just seen this one with the extra `} `...

File: datafiles/streams/atresplayer.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      "attributes": {
+        "geolocked": false},
+    {"name": "lasexta",
+      "title": "La Sexta",
+      "url": "http://www.atresplayer.com/directos/television/lasexta/"},
```
Comment (beardypig) [inline datafiles/streams/atresplayer.json]: And the same here :-)

**Review by beardypig — CHANGES_REQUESTED** [review]
If you fix those two I think this is good to merge :-)

File: datafiles/streams/atresplayer.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    "region": "ES",
+    "language": "esp",
+    "hd": true
+  },
+  "streams": [
+    {"name": "antena3",
+      "title": "Antena 3",
+      "url": "http://www.atresplayer.com/directos/television/antena3/",
+      "attributes": {
+        "geolocked": false},
```
Comment (beardypig) [inline datafiles/streams/atresplayer.json]: Ah, now we are missing `}` before the `,`...

File: datafiles/streams/atresplayer.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    {"name": "lasexta",
+      "title": "La Sexta",
+      "url": "http://www.atresplayer.com/directos/television/lasexta/",
+      "attributes": {
+        "geolocked": false},
```
Comment (beardypig) [inline datafiles/streams/atresplayer.json]: Same here...

**Comment by karlo2105** [comment]
I pushed fixes it it ok now ?

**Review by beardypig — CHANGES_REQUESTED** [review]
I'm going to automate these checks :-)

File: datafiles/streams/atresplayer.json (hunk 7 above)
Comment (beardypig) [reply datafiles/streams/atresplayer.json]: On the `geolocked` line
```
 "geolocked": false}},
```

**Comment by beardypig** [comment]
You can use this website: http://jsonlint.com to validate the json file :-)

**Comment by karlo2105** [comment]
Not sure what to do as everything looks like ok.

```
Error: Parse error on line 14:
...		},		"attributes": {			"geolocked": 
---------------------^
Expecting 'EOF', '}', ',', ']', got ':'
```

**Review by beardypig — APPROVED** [review]

**Comment by beardypig** [comment]
Hope you don't mind that I fixed it :)

**Comment by beardypig** [comment]
Thanks @karlo2105 and congrats 👍

**Comment by karlo2105** [comment]
Thanks for fix I don't mind at all. 
It's just I didn't figure out what was wrong.

**Pull request merged**